src/App.tsx

```diff
@@ -1,24 +1,176 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { deleteTodo, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorList } from './components/ErrorList/ErrorList';
+import {
+  TodoFilter,
+  FilterTodoStatus,
+} from './components/TodoFilter/TodoFilter';
+import { TodoForm, TodoFormProps } from './components/TodoForm/TodoForm';
 
-const USER_ID = 0;
+import { ErrorTypes } from './types/ErrorTypes';
+
+const USER_ID = 6848;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [filter, setFilter] = useState<FilterTodoStatus>(FilterTodoStatus.ALL);
+  const [errors, setErrors] = useState<string>('');
+  const [deletedTodosIds, setDeletedTodosIds] = useState<number[]>([]);
+
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((result) => setTodos(result))
+      .catch(() => setErrors(ErrorTypes.Load));
+  }, []);
+
+  const activeTodos = useMemo(
+    () => todos.filter((todo) => !todo.completed),
+    [todos],
+  );
```

Comment:
It line bigger than 80?
```suggestion
  const activeTodos = useMemo(() => todos.filter((todo) => !todo.completed), [todos]);
```